cmd/csi-driver/csi-driver.go

```diff
@@ -30,11 +31,13 @@ const (
 
 var (
 	// Flag variables for the command options
-	name       string
-	endpoint   string
-	dbServer   string
-	dbPort     string
-	flavorName string
+	name               string
+	endpoint           string
+	dbServer           string
+	dbPort             string
+	flavorName         string
+	nfsMonitor         bool
+	nfsMonitorInterval string
```

Comment:
This all looks great.  But I have to ask, why limit this to monitoring only for nfs pods?  The monitor function below can allow any label to be specified.  Why not pull that out into a separate library that can be extended for other purposes?

Reply:
Yes, i was little cautious to make it generic as clean I/O shutdown is still an issue on failed node and thought coverage with NFS should provide good data. I can make these generic and get labels through ENV.